src/OpenTelemetry.AutoInstrumentation/Loading/Initializers/WcfInitializer.cs

```diff
@@ -25,7 +25,11 @@ internal class WcfInitializer : InstrumentationInitializer
     private readonly PluginManager _pluginManager;
 
     public WcfInitializer(PluginManager pluginManager)
+#if NETFRAMEWORK
+        : base("System.ServiceModel")
+#else
         : base("System.ServiceModel.Primitives")
```

Comment:
Is there any chance that .NET (Core) will not load this library?
I mean in the same scenario as for .NET Fx.

Reply:
`ServiceContractAttribute` is defined in `System.ServiceModel.Primitives` assembly for .NET Core, so you need to reference it to define the contract.